### PR TITLE
Utilities+LibCore: Port some Utilities to LibMain

### PR DESCRIPTION
**Pull request by mjz19910**

### CODE REVIEW

File: Userland/Utilities/CMakeLists.txt [hunk 1]
```diff
@@ -73,9 +73,9 @@ target_link_libraries(comm LibMain)
 target_link_libraries(config LibConfig LibMain)
 target_link_libraries(copy LibGUI LibMain)
 target_link_libraries(cp LibMain)
-target_link_libraries(cpp-lexer LibMain)
-target_link_libraries(cpp-parser LibMain)
-target_link_libraries(cpp-preprocessor LibMain)
+target_link_libraries(cpp-lexer LibCpp LibMain)
```
Comment (bgianfo) [inline Userland/Utilities/CMakeLists.txt:76]: These changes are unrelated to this PR, and look like they came from a commit that was just supposed to sort the entries in this file?

**Review by bgianfo — CHANGES_REQUESTED** [review]

File: Userland/Utilities/CMakeLists.txt (hunk 1 above)
Comment (mjz19910) [reply Userland/Utilities/CMakeLists.txt]: Ok, will continue to be annoyed at the out of order entries. (might open a new pr to sort all cmake target_link_libraries, make it a Everywhere PR)

Comment (bgianfo) [reply Userland/Utilities/CMakeLists.txt]: To be clear, I'm not saying the sorting changes are bad, just that they should be in their own commit, with no unrelated changes mixed in.
